src/test/resources/net/sf/jabref/logic/exporter/BibTeXMLExporterTestInvalidInbook.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<file xmlns="http://bibtexml.sf.net/">
+    <entry id="Mustermann2016">
+        <inbook>
+            <bibtexkey>Mustermann2016</bibtexkey>
```

Comment:
Why is it beeing exported to a new node here but not in the previous test files?